README.md

```diff
@@ -86,14 +86,18 @@ console.log(isValid(fragments)); // display true
     "type": "ISSUER_IDENTITY"
   }
 ]
-
 ```
 
 ## Advanced usage
 
 ### Environment Variables
 
 - `INFURA_API_KEY`: let you provide your own `INFURA` API key.
+- `ALCHEMY_API_KEY`: let you provide your own `ALCHEMY` API key.
```

Comment:
just use PROVIDER_API_KEY for all and figure it out internally, but also continue supporting INFURA_API_KEY for backward compatibility - add a warning when people use that, and ask them to use PROVIDER_API_KEY instead

Reply:
ok